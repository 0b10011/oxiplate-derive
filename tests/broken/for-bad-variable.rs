use oxiplate_derive::Oxiplate;

#[derive(Oxiplate)]
#[oxiplate_inline = "{%- for number in numbers -%}{{ number }} + {{ number_to_add }} = {{ numb + number_to_add }}{% endfor %}"]
struct Data {
    numbers: Vec<i32>,
    number_to_add: i32,
}

fn main() {
    print!(
        "{}",
        Data {
            numbers: vec![19, 89],
            number_to_add: 4,
        }
    );
}
