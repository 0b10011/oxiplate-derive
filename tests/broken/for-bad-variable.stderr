error[E0609]: no field `numb` on type `&Data`
 --> tests/broken/for-bad-variable.rs:4:91
  |
4 | #[oxiplate_inline = "{%- for number in numbers -%}{{ number }} + {{ number_to_add }} = {{ numb + number_to_add }}{% endfor %}"]
  |                                                                                           ^^^^ unknown field
  |
  = note: available fields are: `numbers`, `number_to_add`
